subsys/net/ip/net_pkt.c

```diff
@@ -1868,6 +1868,8 @@ bool net_pkt_is_contiguous(struct net_pkt *pkt, size_t size)
 	if (pkt->cursor.buf && pkt->cursor.pos) {
```

Comment:
There is some extra characters after the signed-off-line in commit "net: ip: net_pkt: Add pkt_cursor_advance before contiguous check"

Reply:
This Commit is not part of this PR. Original PR: 16201